processor/ratelimitprocessor/gubernator_test.go

```diff
@@ -146,50 +145,42 @@ func TestGubernatorRateLimiter_RateLimit(t *testing.T) {
 
 			respErr = errors.New("nope")
 			err = rateLimiter.RateLimit(context.Background(), 1)
-			assert.EqualError(t, err, "error executing gubernator rate limit request: rpc error: code = Unknown desc = nope")
+			assert.EqualError(t, err, errRateLimitInternalError.Error())
 		})
 	}
 }
 
-func TestGubernatorRateLimiter_RateLimit_MetadataKeys(t *testing.T) {
-	server, rateLimiter := newTestGubernatorRateLimiter(t, &Config{
-		Rate:             1,
-		Burst:            2,
-		MetadataKeys:     []string{"metadata_key"},
-		ThrottleBehavior: ThrottleBehaviorError,
-	})
+func TestGubernatorRateLimiter_InternalError(t *testing.T) {
```

Comment:
I don't see we are asserting the internal error, its already done as part of the previous test. Do you feel this test is needed?

Reply:
No, this was a mess I forgot to remove. Thanks for pointing it out.